### PR TITLE
Update avatar-name.tsx

### PR DESCRIPTION
**Pull request by anLpk**
Closes # 

## 📝 Description
Currently, the avatar component has a fallback to generate initials with random and accessible background colours.
But If the users name more than 2 words, initials do not get displayed as expected.

## ⛳️ Current behaviour (updates)

If the user name is `Anil Pak`, it's displayed as `AP`, but If the user name is `Anil Pak Chakra` it will be also displayed as `AP` instead of `AC`

## 🚀 New behavior

With these changes instead of splitting the name as first_name and last_name, we are splitting the string name to the array and taking the first letter from the first and the last word to create initials.

## 💣 Is this a breaking change (Yes/No):

Not sure :)

### CODE REVIEW

File: packages/components/avatar/src/avatar-name.tsx [hunk 1]
```diff
@@ -3,10 +3,10 @@ import { useAvatarStyles } from "./avatar-context"
 import { AvatarOptions } from "./avatar-types"
 
 export function initials(name: string) {
-  const [firstName, lastName] = name.split(" ")
```
Comment (lukemartin) [inline packages/components/avatar/src/avatar-name.tsx:6]: Could change this to:

```ts
const [firstName, ...otherNames] = name.split(" ");
const [lastName] = otherNames.reverse();
```

and keep the rest of the function as-is

Comment (anLpk) [reply packages/components/avatar/src/avatar-name.tsx:6]: Thank Luke! Looks even better!

File: packages/components/avatar/src/avatar-name.tsx [hunk 2]
```diff
@@ -3,7 +3,8 @@ import { useAvatarStyles } from "./avatar-context"
...
+  const [firstName, ...otherNames] = name.split(" ");
+  const [lastName] = otherNames.reverse();
```
Comment (aashutoshrathi) [inline packages/components/avatar/src/avatar-name.tsx:7]: if you just need last element, you should use:
```ts
 const lastName = otherNames.at(-1);
```

**Review by aashutoshrathi — COMMENTED** [review]
left a comment

**Review by aashutoshrathi — APPROVED** [review]

**Pull request merged**